src/test/resources/empty-write-only-model.json

```diff
@@ -0,0 +1,5 @@
+{
+    "propertyE": {
+        "writeOnlyArray": []
+    }
+}
```

Comment:
confused by this test case since it doesn't have any `writeOnlyProperties`

Reply:
Added additional schema input file just to avoid confusion and used the failed scenario as test case.